anynote/api/urls.py

```diff
@@ -5,11 +5,16 @@
     tokenRefreshView,
     tokenObtainPairView,
     tokenVerifyView,
+    notesApiView,
+    noteApiView,
 )
 
 urlpatterns = [
     path("users/", usersApiView, name="usersApi"),
-    path("users/<int:pk>/", userApiView, name="userApi"),
+    path("user/", userApiView, name="userApi"),
+
+    path("notes/", notesApiView, name="notesApi"),
+    path("note/", noteApiView, name="noteApi"),
 
     path("token/", tokenObtainPairView, name="token_obtain_pair"),
     path("token/refresh/", tokenRefreshView, name="token_refresh"),
```

Comment:
Why you have deleted path("users/<int:pk>/") to individual users? It is used to debug user info.
Undo its deletion